### DISCUSSION TITLE
Up arrow in the chat deletes the current unposted message

### DISCUSSION THREAD

**Discussion by SigismondPhilippe** [community/community#78854] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Hi,

I try Copilot in Visual Studio Code.

It is great that the "Ask Copilot" chat bar has a console-like message history, where you press the UP or DOWN arrows to cycle through previous entered questions, but this **instantly deletes the current unposted question** without any way to get it back. It is gone, deleted.

If you were writing an elaborated prompt, "poof". Not cool.

Until it is fixed, we have to compose the prompt somewhere else to not risk sudden deletion...

**Comment by digitarald** [comment, marked as answer]
On the plan for this iteration: https://github.com/microsoft/vscode-copilot-release/issues/661

**Reply by srikgts** [reply]
This problem exists even today - I just ran into it - entered a complex prompt and lost it by accidentally pressing up arrow.

**Reply by JViggiani** [reply]
This problem still exists

**Reply by MiaJohnson** [reply]
PROBLEM STILL EXISTS! 
You've known about this for 3 years? and no fix? 
I am beyond frustrated with this.

**Reply by digitarald** [reply]
The fix was that arrow down brings it back. I assume you don't want history navigation to do anything when text ia already entered?
